Command/DocumentCommand.php

```diff
@@ -0,0 +1,208 @@
+                null,
+                InputOption::VALUE_NONE,
+                'The content will not be signed during the import process'
+            )
+            ->addOption(
+                'force',
+                null,
+                InputOption::VALUE_NONE,
+                'Also treat document in draft mode'
+            )
+            ->addOption(
+                'dont-finalize',
+                null,
+                InputOption::VALUE_NONE,
+                'Don\'t finalize document'
+            )
+            ->addOption(
+                'businessKey',
+                null,
+                InputOption::VALUE_NONE,
+                'Try to identify documents by their business keys'
+            );
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        $arguments = array_values($input->getArguments());
+        array_shift($arguments);
+        list($contentTypeName, $archiveFilename) = $arguments;
+
+        $this->io->title('Make documents');
+        $this->io->section('Checking input');
+
+
+        $contentType = $this->contentTypeService->getByName($contentTypeName);
+        if (!$contentType instanceof ContentType) {
+            $this->io->error(sprintf('Content type %s not found', $contentTypeName));
+            return;
+        }
+
+        if ($contentType->getDirty()) {
+            $this->io->error(sprintf('Content type %s is dirty. Please clean it first', $contentTypeName));
+            return;
+        }
+        $this->contentType = $contentType;
+
+        if (!file_exists($archiveFilename)) {
+            $this->io->error(sprintf('Archive file %s does not exist', $archiveFilename));
+            return;
+        }
+        $this->archiveFilename = $archiveFilename;
+        $this->ready = true;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        if (!$this->ready) {
+            return -1;
+        }
+
+        $options = array_values($input->getOptions());
+        list($bulkSize, $rawImport, $dontSignData, $force, $dontFinalize, $replaceBusinessKey) = $options;
+
+        $signData = !$dontSignData;
+        $finalize = !$dontFinalize;
+
+        $this->io->section(sprintf('Start importing %s from %s', $this->contentType->getPluralName(), $this->archiveFilename));
+
+        $archive = new Archive();
+        $directory = $archive->extractToDirectory($this->archiveFilename);
+
+        $finder = new Finder();
+        $finder->files()->in($directory)->name('*.json');
+        $progress = $this->io->createProgressBar($finder->count());
+        $progress->start();
+        $importerContext = $this->documentService->initDocumentImporterContext($this->contentType, 'SYSTEM_IMPORT', $rawImport, $signData, true, $bulkSize, $finalize, $force);
```

Comment:
```suggestion
        $importerContext = new DocumentImporterContext($this->contentType, 'SYSTEM_IMPORT', $rawImport, $signData, true, $bulkSize, $finalize, $force);
```

Move the logic from initDocumentImporterContext function to the constructor of  that object.